src/webextension/manage/components/item-details.js

```diff
@@ -7,107 +7,59 @@ import PropTypes from "prop-types";
-        notes: "",
-      },
-    };
-  }
-
-  constructor(props) {
-    super(props);
-    this.state = {...props.fields};
-  }
-
-  componentWillReceiveProps(props) {
-    this.setState({...props.fields});
-  }
-
-  render() {
-    const {saveLabel, deleteLabel, onSave, onDelete} = this.props;
-
-    return (
-      <form className={styles.itemDetails} onSubmit={(e) => {
-        e.preventDefault();
-        onSave(this.state);
-      }}>
-        <label>
-          <Localized id="item-details-title">
-            <span>tITLe</span>
-          </Localized>
-          <Input type="text" name="title" value={this.state.title}
-                 onChange={(e) => this.setState({title: e.target.value})}/>
-        </label>
-        <label>
-          <Localized id="item-details-origin">
-            <span>oRIGIn</span>
-          </Localized>
-          <Input type="text" name="origin" value={this.state.origin}
-                 onChange={(e) => this.setState({origin: e.target.value})}/>
-        </label>
-        <label>
-          <Localized id="item-details-username">
-            <span>uSERNAMe</span>
-          </Localized>
-          <Input type="text" name="username" value={this.state.username}
-                 onChange={(e) => this.setState({username: e.target.value})}/>
-        </label>
-        <label>
-          <Localized id="item-details-password">
-            <span>pASSWORd</span>
-          </Localized>
-          <Input type="text" name="password" value={this.state.password}
-                 onChange={(e) => this.setState({password: e.target.value})}/>
-        </label>
-        <label>
-          <Localized id="item-details-notes">
-            <span>nOTEs</span>
-          </Localized>
-          <TextArea name="notes" value={this.state.notes}
-                    onChange={(e) => this.setState({notes: e.target.value})}/>
-        </label>
-        <p></p>
-        <p>
-          <Localized id={saveLabel}>
-            <Button type="submit">sAVe</Button>
-          </Localized>
-          <Localized id={deleteLabel}>
-            <Button type="button" onClick={(e) => onDelete(this.state.id)}>
-              dELETe
-            </Button>
-          </Localized>
-        </p>
-      </form>
-    );
-  }
+export default function ItemDetails({fields, onEdit, onDelete}) {
+  return (
+    <div className={styles.itemDetails}>
+      <div className={styles.field}>
+        <Localized id="item-details-title">
+          <span>tITLe</span>
+        </Localized>
+        <span>{fields.title}</span>
+      </div>
+      <div className={styles.field}>
+        <Localized id="item-details-origin">
+          <span>oRIGIn</span>
+        </Localized>
+        <span>{fields.origin}</span>
+      </div>
+      <div className={styles.field}>
+        <Localized id="item-details-username">
+          <span>uSERNAMe</span>
+        </Localized>
+        <span>{fields.username}</span>
+      </div>
+      <div className={styles.field}>
```

Comment:
To confirm, the plan is removing password from display here in this PR and adding it back (obfuscated) as part of #53, right? (knowing its also available during edit mode)